lib/lumberjack/server.rb

```diff
@@ -214,13 +226,14 @@ def initialize(fd)
       super()
       @parser = Parser.new
       @fd = fd
+      @closed = Concurrent::AtomicBoolean.new
```

Comment:
Is there a way we can close all connections? If we ask the lumberjack input to stop, and it only stops the acceptor thread, it will leave all the currently-open connections still running.

Put a different way, I'm not seeing where/how a `stop` from logstash via the lumberjack input will cause all the connections to stop as well.